onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -325,7 +325,7 @@ class GpuDFTOperator : public WRL::Base<IMLOperatorKernel>
             std::array<uint32_t, 4> reshapedInputStrides = { 1, 1, 1, 1 };
             std::array<uint32_t, 4> reshapedOutputStrides = { 1, 1, 1, 1 };
             std::array<uint32_t, 4> temporaryStrides = { 1, 1, 1, 1 };
-            for (int i = static_cast<int>(reshapedInputSize.size()) - 2; i >= 0; i--)
+            for (uint64_t i = static_cast<uint64_t>(reshapedInputSize.size()) - 2; i >= 0; i--)
```

Comment:
Dead loop? i is always >=0 since it is unsigned.

Reply:
Fixed